setup.py

```diff
@@ -17,55 +16,7 @@ def get_version():
 }
 extras["all"] = sum(extras.values(), [])
 
-install_requires = open("requirements.txt").read().strip().split("\n")
-
-setup(name = "uproot",
-      packages = find_packages(exclude = ["tests"]),
-      scripts = [],
-      version = get_version(),
-      author = "Jim Pivarski",
-      author_email = "pivarski@princeton.edu",
-      maintainer = "Jim Pivarski",
-      maintainer_email = "pivarski@princeton.edu",
-      description = "ROOT I/O in pure Python and NumPy.",
-      long_description = open("README.md").read(),
-      long_description_content_type = "text/markdown",
-      url = "https://github.com/scikit-hep/uproot4",
-      download_url = "https://github.com/scikit-hep/uproot4/releases",
-      license = "BSD 3-clause",
-      test_suite = "tests",
-      python_requires = ">=2.6, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*",
```

Comment:
Wow, I totally read this as 2.7! I must have copied and pasted from the scikit-hep developer pages and have been looking at that! So yours was consistent and correct. I thought they were mismatched.